docs/resources/job.md

```diff
@@ -158,7 +158,7 @@ You can invoke Spark submit tasks only on new clusters. **In the `new_cluster` s
 
 ### spark_python_task Configuration Block
 
-* `python_file` - (Required) The URI of the Python file to be executed. [databricks_dbfs_file](dbfs_file.md#path) and S3 paths are supported. This field is required.
+* `python_file` - (Required) The URI of the Python file to be executed. [databricks_dbfs_file](dbfs_file.md#path), cloud file URIs (`s3:/`, `adls:/`, `gcs:/`) and workspace paths are supported. For python files stored in the Databricks workspace, the path must be absolute and begin with `/Repos`. This field is required.
```

Comment:
The URI is written here as `adls:/` but when accessing Azure Data Lake Storage the most common way is with `abfss:/`. Shall we update it? 

And maybe the same for `s3:/` and `s3a:/`?